config.js

```diff
@@ -375,7 +331,7 @@ if (argv.botid || argv.bot || argv.id || argv.minrankedhandicap || argv.maxranke
         return false;
     }
 
-    if (argv.ban) {
+    if (argv.ban && !argv.banranked && !argv.banunranked) {
```

Comment:
No need for this check on `!argv.banranked && !argv.banunranked`. just append `--ban` to both lists and append the other two to their own list.

Reply:
same !